doc/tutorials/intermediate/parameters.md

```diff
@@ -0,0 +1,394 @@
+
+Try replacing `pn.pane.DataFrame` with `pn.panel` to display an interactive component with widgets.
+
+:::{dropdown} Solution
+
+```{pyodide}
+import pandas as pd
+import panel as pn
+import numpy as np
+
+pn.extension()
+
+data_url = 'https://assets.holoviz.org/panel/tutorials/turbines.csv.gz'
+df = pn.cache(pd.read_csv)(data_url)
+
+dfrx = pn.rx(df)
+
+cols  = pn.widgets.MultiChoice(
+    options=df.columns.to_list(), value=['p_name', 't_state', 't_county', 'p_year', 'p_cap'], height=300
+)
+nrows = pn.widgets.IntSlider(start=5, end=20, step=5, value=15, name='Samples')
+style = pn.rx('color: white; background-color: {color}')
+color = pn.widgets.ColorPicker(value='darkblue', name='Highlight color')
+
+def highlight_max(s, props=''):
+    if s.dtype.kind not in 'f':
+        return np.full_like(s, False)
+    return np.where(s == np.nanmax(s.values), props, '')
+
+styled_df = dfrx[cols].sample(nrows).style.apply(highlight_max, props=style.format(color=color), axis=0)
+
+pn.panel(styled_df).servable()
+```
+
+:::
+
+### Exercise: Scale the Font Size
+
+Write a small app where you can scale the `font-size` of a `Markdown` pane with another widget, e.g. an `IntSlider`. The `font-size` can be set using the `styles` parameter.
+
+:::{hint}
+
+- The `styles` parameter only accepts dictionaries of **strings** but `IntSlider` returns `int` types.
+- The `font-size` value should be a string value in pixels, eg. `"15px"` is a valid `font-size`.
+
+:::
+
+:::{dropdown} Solution 1: Reactive String Formatting
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+intslider = pn.widgets.IntSlider(value=10, start=5, end=100, step=10, name="Font Size")
+
+styles_rx = {"font-size": pn.rx("{value}px").format(value=intslider)}
+markdown = pn.pane.Markdown("We love dataviz!", styles=styles_rx)
+pn.Column(intslider, markdown).servable()
+```
+
+:::
+
+:::{dropdown} Solution 2: Reactive Function
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+intslider = pn.widgets.IntSlider(value=10, start=5, end=100, step=10, name="Font Size")
+
+def styles(font_size):
+    return {"font-size": f"{font_size}px"}
+
+styles_rx = pn.rx(styles)(intslider)
+markdown = pn.pane.Markdown("We love dataviz!", styles=styles_rx)
+pn.Column(intslider, markdown).servable()
+```
+
+:::
+
+::::{dropdown} Solution 3: Bound Function
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+intslider = pn.widgets.IntSlider(value=10, start=5, end=100, step=10, name="Font Size")
+
+def styles(font_size):
+    return {"font-size": f"{font_size}px"}
+
+styles_bn = pn.bind(styles, font_size=intslider)
+markdown = pn.pane.Markdown("We love dataviz!", styles=styles_bn)
+pn.Column(intslider, markdown).servable()
+```
+
+:::{note}
+
+`pn.bind` is the predecessor of `pn.rx`. We recommend using `pn.rx` over `pn.bind` as it's much more flexible and efficient. We include this example because you will find lots of examples in the Panel documentation and in the Panel community using `pn.bind`.
```

Comment:
Or rather `pn.rx is the successor of `pn.bind`.? This is good to know by the way, I did not realize this before. Is there a separate section somewhere regarding this in the docs (starting with the pn.bind) documentation? 

Something that says at the top: We recommend pn.rx instead of pn.bind from now on? And a link to a migration tutorial that covers most cases of how best to change existing code that uses pn.bind?

Reply:
I think I will have to update that. We discussed this at HoloViz meeting today and we are not really ready to recommend transition yet.